app/models/card.rb

```diff
@@ -1,4 +1,20 @@
 class Card < ActiveRecord::Base
-  validates :translated_text, :original_text,  presence: true
+  validates :translated_text, :original_text, presence: true
   validates_with CardTextFieldsDifferenceValidator
+  scope :relevant_for_today, -> { where("review_date <= ?", Time.now) } 
+  scope :random_sorted, -> { order("RANDOM()") } 
+
+  def review(user_answer)
+    user_answer.strip!
+    user_answer.downcase!
+    
+    if translated_text.downcase == user_answer 
+      days_to_move = 3
+      update_attributes( { review_date: (Time.now.midnight + days_to_move.day) } )
```

Comment:
Line is too long. [82/80]<br>Redundant curly braces around a hash parameter.<br>Space inside parentheses detected.